com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -640,7 +720,7 @@ void GenerateShaderPass(int targetIndex, PassDescriptor pass, ActiveFields activ
 
             using (var propertyBuilder = new ShaderStringBuilder())
             {
-                propertyCollector.GetPropertiesDeclaration(propertyBuilder, m_Mode, m_GraphData.graphDefaultConcretePrecision);
+                subShaderProperties.GetPropertiesDeclarationForTarget(propertyBuilder, m_Mode, m_GraphData.graphDefaultConcretePrecision, target);
```

Comment:
replace the per-pass property declaration with the subshader property declaration